Conda build for Jupytext 1.2.2 is in error

As `pkg_ressources` imported in `pandoc.py` in not an explicit dependency.

This may be a side effect of #310 .

Below is an extract of the logs:
```bash
Packaging jupytext
INFO:conda_build.build:Packaging jupytext
INFO conda_build.build:build(1570): Packaging jupytext
INFO:conda_build.build:Packaging jupytext-1.2.2-0
INFO conda_build.build:bundle_conda(891): Packaging jupytext-1.2.2-0
Packaging jupytext-1.2.2-0
number of files: 43
Fixing permissions
Packaged license file/s.
Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
WARNING:conda_build.build:Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
WARNING conda_build.build:bundle_conda(1042): Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
TEST START: /home/conda/feedstock_root/build_artifacts/noarch/jupytext-1.2.2-0.tar.bz2
Adding in variants from /tmp/tmphdeivyu7/info/recipe/conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from /tmp/tmphdeivyu7/info/recipe/conda_build_config.yaml
INFO conda_build.variants:_combine_spec_dictionaries(188): Adding in variants from /tmp/tmphdeivyu7/info/recipe/conda_build_config.yaml
Renaming work directory,  /home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/work  to  /home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/work_moved_jupytext-1.2.2-0_noarch
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: /home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho


The following NEW packages will be INSTALLED:

    _libgcc_mutex:    0.1-main             defaults   
    bzip2:            1.0.8-h516909a_0     conda-forge
    ca-certificates:  2019.6.16-hecc5488_0 conda-forge
    decorator:        4.4.0-py_0           conda-forge
    ipython_genutils: 0.2.0-py_1           conda-forge
    jsonschema:       2.6.0-py37_1002      conda-forge
    jupyter_core:     4.4.0-py_0           conda-forge
    jupytext:         1.2.2-0              local      
    libffi:           3.2.1-he1b5a44_1006  conda-forge
    libgcc-ng:        9.1.0-hdf63c60_0     defaults   
    libstdcxx-ng:     9.1.0-hdf63c60_0     defaults   
    nbformat:         4.4.0-py_1           conda-forge
    ncurses:          6.1-hf484d3e_1002    conda-forge
    openssl:          1.1.1c-h516909a_0    conda-forge
    python:           3.7.3-h33d41f4_1     conda-forge
    pyyaml:           5.1.2-py37h516909a_0 conda-forge
    readline:         8.0-hf8c457e_0       conda-forge
    six:              1.12.0-py37_1000     conda-forge
    sqlite:           3.29.0-hcee41ef_1    conda-forge
    tk:               8.6.9-hed695b0_1002  conda-forge
    traitlets:        4.3.2-py37_1000      conda-forge
    xz:               5.2.4-h14c3975_1001  conda-forge
    yaml:             0.1.7-h14c3975_1001  conda-forge
    zlib:             1.2.11-h516909a_1005 conda-forge

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
export PREFIX=/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/test_tmp
import: 'jupytext'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/test_tmp/run_test.py", line 2, in <module>
    import jupytext
  File "/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/jupytext/__init__.py", line 3, in <module>
    from .jupytext import read, write, readf, writef, writes, reads
  File "/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/jupytext/jupytext.py", line 12, in <module>
    from .formats import _VALID_FORMAT_OPTIONS
  File "/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/jupytext/formats.py", line 19, in <module>
    from .pandoc import pandoc_version, is_pandoc_available
  File "/home/conda/feedstock_root/build_artifacts/jupytext_1567359616966/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/jupytext/pandoc.py", line 8, in <module>
    from pkg_resources import parse_version
ModuleNotFoundError: No module named 'pkg_resources'
Tests failed for jupytext-1.2.2-0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for jupytext-1.2.2-0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING conda_build.build:tests_failed(2230): Tests failed for jupytext-1.2.2-0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: jupytext-1.2.2-0.tar.bz2
##[error]Bash exited with code '1'.
##[section]Finishing: Run docker build
```

--- a/jupytext/pandoc.py
+++ b/jupytext/pandoc.py
@@ -5,7 +5,6 @@ import subprocess
 import tempfile
 # Copy nbformat reads and writes to avoid them being patched in the contents manager!!
 from nbformat import reads as ipynb_reads, writes as ipynb_writes
-from pkg_resources import parse_version
 
 
 class PandocError(OSError):
@@ -44,6 +43,12 @@ def is_pandoc_available():
 def pandoc_version():
     """Pandoc's version number"""
     version = pandoc(u'--version').splitlines()[0].split()[1]
+
+    try:
+        from pkg_resources import parse_version
+    except ImportError:
+        raise PandocError('Please install pkg_resources')
+
     if parse_version(version) < parse_version('2.7.2'):
         raise PandocError('Please install pandoc>=2.7.2 (found version {})'.format(version))
 

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -5,6 +5,7 @@ codecov
 notebook
 jupyter_client
 nbconvert
+setuptools
 
 # Python 2
 pathlib

